src/styles/empty-view.module.scss

```diff
@@ -0,0 +1,14 @@
+.wrapper {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    top: 0;
+    left: 0;
+    right: 0;
+    width: 100%;
+    height: 100%; 
+    overflow: auto; 
+    position: fixed;
+    z-index: 2;
+    background-color: #fff;
```

Comment:
Please use color from themes

Reply:
should theme be applied here since it will always be white?